api/test/integration/test_agent_DELETE_endpoints.tavern.yaml

```diff
@@ -537,6 +537,9 @@ stages:
 ---
 test_name: DELETE /agents
 
+marks:
```

Comment:
Just to not forget about it, we should add a bullet point to fix tests marked as xfail in the technical debt issue

Reply:
Done